recipes/centroid_rna_package/meta.yaml

```diff
@@ -0,0 +1,33 @@
+# building from source requiers also boost >= 1.4
+# this recipe uses working precompiled binaries
+package:
+  name: centroid_rna_package
+  version: 0.0.15
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  sha1: e4318ffcd3bd4419a68c273fd924ec17a366c073
+  url: https://github.com/satoken/centroid-rna-package/releases/download/v0.0.15/centroid-rna-package-0.0.15-linux-x86_64.zip
+
+requirements:
+  run:
+    - viennarna >=1.8
+
+test:
+  requires:
```

Comment:
remove the requirement and add a real test, for example exectuing your tool with `--help` or `--version`

Reply:
In conda documentation I found that the test are either executed with command inside meta.yaml or in form of run_test.[sh/py...].

I've initially not included the test in meta.yaml because the exit value of the --help is 1, so the test fails. I've searched on how this is usually handled and came with two options: 
1) check the exit value
I can't check exit value, because it is always 1, even when the program fails. 
2) check the return string
So I need to check the return string rather then exit code. I've written the check in python and that’s where the python test dependency comes in. The python is otherwise not needed.

I agree that it might not be the best idea and I propose to fix it with test in meta.yaml and checking the text with grep (assumes grep avalible).